http/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/auth/CloudServiceAuthTokenProviderSpec.scala

```diff
@@ -27,7 +27,8 @@ class CloudServiceAuthTokenProviderSpec
     val result = for {
       now <- IO.realTimeInstant
     } yield {
-      tokens.push(CloudToken(secondTokenValue, now.plusSeconds(10)))
+      // set secondTokenValue to greater than the tokenCacheTimeInMilli which is 30 minutes
+      tokens.push(CloudToken(secondTokenValue, now.plusSeconds(40 * 60)))
```

Comment:
30 * 60 + 10?

Reply:
Sure, that might be clearer. Though 40 minutes covers it ;)